Python runner in quickstart guide (#3911)
cocotb/cocotb: docs/source/quickstart.rst

    make SIM=questa

.. _quickstart_using_runners:

Using runners
=============

.. warning::
    Python runners and associated APIs are an experimental feature and subject to change.

An alternative to the ``Makefile`` is to use the Python Test Runners, or "runner" for short.

> Let's define a runner in the ``test_my_design_runner`` function directly in our test file:

Reviewer: Let's not recommend this as an idiom, it has tons of pitfalls.

```suggestion
Let's define a runner for ``test_my_design_runner``:
```
